includes/express-checkout/class-wc-payments-express-checkout-button-handler.php

```diff
@@ -93,7 +93,6 @@ public function init() {
 		add_filter( 'woocommerce_cart_needs_shipping_address', [ $this, 'filter_cart_needs_shipping_address' ], 11, 1 );
 		add_action( 'wp_enqueue_scripts', [ $this, 'scripts' ] );
 		add_filter( 'woocommerce_gateway_title', [ $this, 'filter_gateway_title' ], 10, 2 );
-		add_action( 'woocommerce_checkout_order_processed', [ $this->express_checkout_helper, 'add_order_payment_method_title' ], 10, 2 );
```

Comment:
This hook was not used in the payment flow.
The `woocommerce_checkout_order_processed` action is invoked on the "classic" checkout.
The older implementation of the ECE/PRB leveraged the classic checkout.
But the newer one uses the Store API. And the Store API doesn't call this action.